packages/react-core/src/components/FileUpload/examples/FileUploadTextWithRestrictions.tsx

```diff
@@ -62,6 +57,11 @@ export const TextFileUploadWithRestrictions: React.FunctionComponent = () => {
           validated={isRejected ? 'error' : 'default'}
           browseButtonText="Upload"
         />
+        <HelperText>
+          <HelperTextItem variant={isRejected ? 'error' : 'default'}>
+            {isRejected ? 'Must be a CSV file no larger than 1 KB' : 'Upload a CSV file'}
+          </HelperTextItem>
+        </HelperText>
```

Comment:
IMHO, we want to keep the `.pf-c-form__helper-text` because that is styling where in a form we want to place the helper text. Similarly, we have date picker helper text and progress helper text that are customizing where the helper text goes within those components.

Reply:
Re-added the wrapper. My question now is whether we want to strip out the extra functionality that `FormHelperText` includes which is covered by `HelperText`, since it is only used for spacing now?

Reply:
I think we should still have the `helperText` prop.  That way we can wrap the Helper text internally insteas of relying on the consumer to do it.  That is what we do with the progress component.  We could then remove the `FormHelperText` component entirely.

Reply:
This would require the `isHelperTextBeforeField` prop to remain as well so a user could still control the helper text position. I think it would be cleaner to have all the props fully removed and let the user define what they want in `children` but LMK and I can update to re-include those props.

Reply:
That is a good point Katie.  If we decide to go the route of adding `isHelperTextBeforeField` back, maybe we group that together with hhe helper text in a structure.  I think there are pros and cons to either approach...

Reply:
I do feel that if we leave it up to the consumer to add the `FormHelperText` wrapper themselves, it will not always happen.  At quick glance you would not know that there is supposed to be a different sized gap.

Reply:
Yes, a blurb would be helpful.